lib/IRGen/IRGenSIL.cpp

```diff
@@ -28,6 +28,8 @@
 #include "swift/AST/ParameterList.h"
 #include "swift/AST/Pattern.h"
 #include "swift/AST/SubstitutionMap.h"
+// SWIFT_ENABLE_TENSORFLOW
```

Comment:
Clarification question: Why do we have #ifdefs in some places and comment in other places?

Reply:
#ifdefs is intended to makes sure our branch can still be compiled without TF support. This is probably not consistently enforced though. 

I'm also not sure if that's necessary. e.g. As a strategy to upstream our code, it might be sufficient to always include the TF related bits into the compiled binary, but protect the TF code path under some compiler flags. We can discuss this more outside this PR scope.